src/module.c

```diff
@@ -151,37 +258,42 @@ int TSDB_queryindex(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     return REDISMODULE_OK;
 }
 
-int TSDB_rangebylabels(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+int TSDB_mrange(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     RedisModule_AutoMemory(ctx);
     api_timestamp_t start_ts, end_ts;
     api_timestamp_t time_delta = 0;
-    RedisModuleString * aggTypeStr = NULL;
-    AggregationClass *aggObject = NULL;
-    int agg_type = 0;
 
     if (argc < 4)
         return RedisModule_WrongArity(ctx);
-    int query_count = argc - 1 - 2; // 1 is for the command, 2 is for start_ts and end_ts
+    Series fake_series;
```

Comment:
good practice to initialize to 0; Series = fake_series {0}; unless the structure is very large.